scipy-1.0/paper.tex

```diff
@@ -1040,7 +1040,7 @@ \section*{Discussion}
 
 Nevertheless, SciPy continually strives to improve.
 The SciPy Roadmap\cite{SciPy_roadmap_1,SciPy_roadmap_dev}, summarized in Table~\ref{table:roadmap}, is a
-continuously-updated document
+continuously updated document
```

Comment:
```suggestion
continually updated document
```
Moreover, it should by _continually_ updated, as it is updated frequently as opposed to without ceasing.

Reply:
Please commit this suggestion to your branch and I will merge this PR.